src/proxy/keepassxc-proxy.cpp

```diff
@@ -55,13 +55,28 @@ void catchUnixSignals(std::initializer_list<int> quitSignals)
         sigaction(sig, &sa, nullptr);
     }
 }
+#else
+#include <windows.h>
```

Comment:
This should be at the beginning of the file.

Reply:
Technically it is. The top #ifdef block includes headers. I'll add a blank space after it though since that is against convention.